### DISCUSSION TITLE
직접 만든 클래스 인스펙터에 노출 문제와  ZepetoScriptableObject 사용 문제. 서버 저장소 관련

### DISCUSSION THREAD

**Discussion by watermeat2020** [naverz/zepeto-studio-kor#1214] [5. Q&A]
1. 직접 만든 클래스 인스펙터 노출 관련해서 논의 중인 듯 한데 #230 언제 쯤 업데이트 되나요?

2. 위 문제로 인해 ScriptableObject가 대량의 데이터를 사용하는 컨테이너 인데 본래 기능을 못하는 듯합니다. 
    혹시 추천하시는 데이터( 대량의 템플릿 데이터 ) 처리 방식이 있나요?

3.  #691 Server DataStorage  서버에 저장할 수 있는 유저 당 용량이 10kb 이라고 하는데 key 값도 포함인가요? 
     valuse 값만 생각하면 될까요? 
     10kb => 81920bit / 64bit = 1,280    number 타입으로 1280개로 생각 하면 되는건가요?
     데이터 초과 했을 경우 어떻게 되나요? 
     데이터 모니터링이라던지 전체 초기화 할 수 있는 방법이 있나요?

힘드네요 도와주세요....

**Comment by zlmrzlo** [comment]
https://studio.zepeto.me/guides/scriptableobject
이 링크도 한 번 참조해보셨나요?

**Reply by watermeat2020** [reply]
네 타입스크립트 기본 타입은 인스펙터에 나오는데 직접 만든 클래스는 나오질 않아요.
직접 만든 클래스를 멤버 변수로 선언 했을 때 입니다.

**Reply by zlmrzlo** [reply]
클래스 타입이 인스펙터에 나오지는 않더라도 사용은 가능한 걸로 알고 있어요.

**Reply by watermeat2020** [reply]
런타임 일 때 접근 가능하고 생성 하는 것 보다, 에디터 상에서 ScriptableObject.asset 에 데이터를 넣을 수 있어야 의미가 있다고 생각합니다.
